macros/fivetran/base/fivetran_fb_campaigns.sql

```diff
@@ -0,0 +1,30 @@
+{% macro fivetran_fb_ads_campaigns() %}
+
+    {{ adapter_macro('facebook_ads.fivetran_fb_ads_campaigns') }}
+
+{% endmacro %}
+
+
+{% macro default__fivetran_fb_ads_campaigns() %}
```

Comment:
does this actually need two different versions? they are nearly identical...

Reply:
@ChrisLawliss this still has two separate versions and the only difference is the `nullif()`s.